plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/xslt/config/XSLTGeneratorConstants.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.developerstudio.datamapper.diagram.xslt.config;
+
+public final class XSLTGeneratorConstants {
+
+    private XSLTGeneratorConstants() {
+    }
+
+    // constants
+    public static final String STRING_TYPE = "string";
+    public static final String OBJECT_TYPE = "object";
+    public static final String ARRAY_TYPE = "array";
+    public static final String NUMBER_TYPE = "number";
+    public static final String BOOLEAN_TYPE = "boolean";
```

Comment:
Shall we define types (relation to nodes in .datamapper file?)
i.e array type is a node having child nodes.